Source/Core/UICommon/AutoUpdate.cpp

```diff
@@ -0,0 +1,139 @@
+// Copyright 2018 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#include "UICommon/AutoUpdate.h"
+
+#include <picojson/picojson.h>
+
+#include "Common/CommonPaths.h"
+#include "Common/FileUtil.h"
+#include "Common/HttpRequest.h"
+#include "Common/Logging/Log.h"
+#include "Common/StringUtil.h"
+#include "Common/scmrev.h"
+#include "Core/ConfigManager.h"
+
+#ifdef _WIN32
+#include <Windows.h>
+#endif
+
+namespace
+{
+bool SystemSupportsAutoUpdates()
+{
+#ifdef _WIN32
+  return true;
+#else
+  return false;
+#endif
+}
+
+#ifdef _WIN32
+
+const char* UPDATER_FILENAME = "Updater.exe";
```

Comment:
These should likely use the array form:

```cpp
const char fart_str[] = "fart";
```
or be `constexpr const char*`, so it's not storing the pointer in the executable along with the string data (not that it matters much here, but I figured why not).